lib/gateway/actionParams.js

```diff
@@ -0,0 +1,30 @@
+const log = require('../logger').policy;
+class ActionParams {
+}
+
+// TODO: this may be not the best way,
+// but it seems to be the only one to allow reuse this code in plugins without requiring SDK
+ActionParams.prototype.getCommonAuthCallback = function (req, res, next) {
+  return (err, user, info) => {
+    // passThrough allows auth to fail
+    // (hoping that next auth policy will be able to do auth)
+    if (!user && (this.passThrough || this.passthrough)) {
```

Comment:
I would leave both same way it is `path` vs `paths` 
And adding it to schema is ok

Reply:
@XVincentX @DrMegavolt I understand both positions.  We allow `path` and `paths` because we allow either a string or an array for that value.  The flexibility in type creates confusion, so we allow both.  This change with allowing both `passthrough` and `passThrough` has a slightly different motivation than `path`/`paths`, but it does complicate the schema.  I'm inclined to just pick one at this time and consider adding a helper later should this prove to be an issue.

The question is... which do we pick?  I think of `Passthrough` as a single word when it's a noun, and frankly, I find it confusing that the Node stream is named `PassThrough`.  In this case, however, I think we might actually be using it as a verb phrase.  In that case, I'd opt for `passThrough`.  This relates to user experience, and therefore, I suggest we also get @altsang's opinion.

Reply:
I'm all for having single option in schema. I've added the property to each creds.(single prop) 
When I was playing with thing I noticed that sometimes I forget to write it camel case. that is why I added it.

Reply:
I'd go with `passthrough` but it's a personal preference and I do not neither have good arguments to support it.

What I care about is having a single way to express something, for a lot of reasons.
The first one that's coming into my mind is the difficulty of representing such situation with a JSON Schema:

```javascript
{
  type: 'object',
    properties: {
    passthrough: {
      type: 'string',
          },
    passThrough: {
      type: 'string'
    }
  },
  oneOf: [{ required: ['passthrough'] }, { required: ['passThrough'] }]
}
```

There might be problems when used with default values as well

Reply:
`passThrough` = consistent with most authentication docs I’ve seen, and is a verb phrase

Reply:
A good, single way is always preferable. Otherwise, it starts to feel a bit like the Wild West. We've only made exceptions for user experience, and it's probably worth revisiting that configuration anomaly.

Reply:
So leave only passThrough and don't have fallback in code ?